src/map/lua/lua_ability.h

```diff
@@ -44,9 +44,11 @@ class CLuaAbility
     uint16 getID();
     int16  getMsg();
     uint16 getRecast();
+    uint16 getRecastID();
     uint16 getRange();
     auto   getName() -> const char*;
     uint16 getAnimation();
+    uint16 getAddType();
```

Comment:
What is add type? Additional type? Could use full version of the name

Reply:
addType is the same as MobSkill's addType, the intent here was to make the 2 types as close to the same as possible so if someone understands mobskill they should understand petskill pretty easily too. I can add a comment about it

Reply:
it being called addType is really the only thing that snags for me, but don't have a naming suggestion either. good enough I guess.